### DISCUSSION TITLE
Import/Export Global Settings fo CSS

### DISCUSSION THREAD

**Discussion by thisbit** [builderius/builderius#41] [Ideas]
I would like to be able to have a starter settings when I build the site. This would involve css variables, defined on the global scope, and a global custom css.

It would be great if this could be exported/imported unrelated to other template work being done.
Use case other then this starting the sitebuild one, might be to test different [typescales](https://type-scale.com/ ) on the same layout.

I build the layout, define with type related css vars() defined. I export them and give this export a name (type-scale-1-200) then I change the values to the new values and export this as type-scale-1-333 and repeat this for different typescales. Then I can have the layout prepared. I import  type-scale-1-200, not happy, import type-scale-1-333 ... and so on.

It would be great feature to test differnt typography settings.

**Comment by den-v** [comment]
@thisbit there is possibility to import/export global settings(but all settings - css and non css). Please open revisions -> change entity from "Template" to "Global Settings". 
Import button you should see easily:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/8387898/199967883-83fee753-b973-4f42-8cb6-3faf0ef53bb2.png">
Export button you will see when you hover over revision:
<img width="297" alt="image" src="https://user-images.githubusercontent.com/8387898/199968195-a88f5f2a-6eac-4afb-b5d1-99806ab89496.png">
